cardstack/src/screens/BuyPrepaidCard/Components.tsx

```diff
@@ -72,15 +70,17 @@ export const CardContent = ({
         >
           $ {amount}
         </Text>
-        <Text
-          color={isSoldOut ? 'buttonSecondaryBorder' : subtitleColor}
-          fontSize={14}
-          textAlign="center"
-          weight="regular"
-        >
-          {`\n`}
-          {isSoldOut ? 'SOLD OUT' : `${faceValue} SPEND`}
-        </Text>
+        {isSoldOut && (
+          <Text
+            color={isSoldOut ? 'buttonSecondaryBorder' : subtitleColor}
```

Comment:
This condition is unnecessary now that we only show this text when `isSoldOut` is true.

 ```suggestion
            color="buttonSecondaryBorder"
```